src/http.ts

```diff
@@ -208,6 +209,13 @@ interface CreateBundleOptions extends ConnectionOptions {
 export async function createBundle(
   options: CreateBundleOptions,
 ): Promise<Result<RemoteBundle, CreateBundleErrorCodes>> {
+  let payloadBody;
+  if (options.base64Encoding) {
+    const payloadBuffer = Buffer.from(JSON.stringify(options.files));
+    payloadBody = payloadBuffer.toString('base64');
```

Comment:
Question: Do you think this could have a negative effect on performance? Like does it slow down file upload of big repos by 300%?

Reply:
There will be a negative impact on performance. I'll get some comparative tests and add here.

Reply:
As discussed - we'll get a separate activity raised to evaluate performance